src/bootstrap/bootstrap.py

```diff
@@ -678,10 +678,10 @@ def update_submodule(self, module, checked_out, recorded_submodules):
 
         print("Updating submodule", module)
 
-        run(["git", "submodule", "-q", "sync", "--progress", module],
+        run(["git", "submodule", "-q", "sync", module],
```

Comment:
Rather than dropping it entirely, maybe it should only be removed from the `sync` line? Per manpage `--progress` is only supported for `add` and `update`, so it should not be added for `sync`.